### ISSUE TITLE
What the Output of the Program?

### ISSUE THREAD

**Issue by Wakeelfahmed** [guochengqian/Magic123#39]
I want to know the 3D object output of the program. is it .obj?

Not mentioned anywhere

**Comment by guochengqian** [comment]
yes. the nerf model ,and the obj. We will release obj once I go back Snap. Now I lose my access to Snap's resources.

**Comment by Jacky-MYQ** [comment]
> yes. the nerf model ,and the obj. We will release obj once I go back Snap in Dec. Now I lose my access to Snap's resources.
I am looking forward to it!!! I want to it can generate a obj too!!

**Comment by guochengqian** [comment]
Answered in v0.1.1 docs. Magic123 can export a textured .obj when run with --save_mesh; the .mp4 outputs are render previews, and the .obj/.mtl/texture files are the 3D object assets.
